hooks/pre-checkout

```diff
@@ -120,6 +120,7 @@ main() {
     repo_dir="$(print_repo_checkout_path "${MAIN_REPO}")"
     export BUILDKITE_PLUGIN_GITHUB_FETCH_S3_URL="${BUILDKITE_PLUGIN_GITHUB_FETCH_S3_URL}/${repo_dir}"
     export BUILDKITE_REPO="${MAIN_REPO}"
+    buildkite-agent meta-data set "MAIN_REPO" "${MAIN_REPO}"
```

Comment:
should we instead set this metadata always (after these if statements)? and maybe with a less specific name (for example `REPO_CHECKOUT`).

My main concern is that we would be setting this for a very specific use case, it may be better to have this more generic and always set the repo that was actually checked out, so we can use it in any case we need.

Reply:
updated